specs/Modal.spec.js

```diff
@@ -271,6 +271,34 @@ describe('State', () => {
     expect(!isBodyWithReactModalOpenClass()).toBeTruthy();
   });
 
+  it('supports adding/removing multiple document.body classes', () => {
+    renderModal({
+      isOpen: true,
+      bodyOpenClassName: 'A B C'
+    });
+    expect(document.body.classList.contains('A', 'B', 'C')).toBe(true);
+    unmountModal();
+    expect(document.body.classList.contains('A', 'B', 'C')).toBe(false);
+  });
+
+  it('does not remove shared classes if more than one modal is open', () => {
+    renderModal({
+      isOpen: true,
+      bodyOpenClassName: 'A'
+    });
+    renderModal({
+      isOpen: true,
+      bodyOpenClassName: 'A B'
+    });
+
+    expect(isBodyWithReactModalOpenClass('A B')).toBe(true);
+    unmountModal();
+    expect(isBodyWithReactModalOpenClass('A B')).toBe(false);
+    expect(isBodyWithReactModalOpenClass('A')).toBe(true);
+    unmountModal();
+    expect(isBodyWithReactModalOpenClass('A')).toBe(false);
```

Comment:
Just to follow the current code style.

My opinion: I'd like to see something like `expect(!something_that_is_false);`, because every expectation ends up been a true statement.

`F = F => T`